nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -452,6 +465,13 @@ define([
             cellTabs.setState(state.tab);
             controlPanel.setActionState(state.action);
             fileTypePanel.updateState(state.fileType);
+            updateJobState();
+            // TODO: add in the FSM state
+            FSMBar.showFsmBar({
+                ui: ui,
+                state: {},
+                job: testDataModel.getItem('exec.jobState'),
+            });
```

Comment:
add updating the job status line and the FSM debug bar to the list of tasks performed by `updateState`